public/app/features/alerting/unified/components/rule-editor/notificaton-preview/NotificationRoute.tsx

```diff
@@ -102,7 +106,8 @@ function NotificationRouteHeader({
 
 interface NotificationRouteProps {
   route: RouteWithPath;
-  receiver: Receiver;
+  receiver?: Receiver;
+  receiverName?: string;
```

Comment:
When would the receiver name be empty? Is this for `null` contact points?

Reply:
The existing types says that receiver on a Route can be null, unfortunately:
https://github.com/grafana/grafana/blob/alerting/contact-point-details-rbac/public/app/plugins/datasource/alertmanager/types.ts#L119

Reply:
https://github.com/grafana/grafana/pull/104639/files#r2068195873